### PR TITLE
Make the project runnable out of the box

### PR DESCRIPTION
**Pull request by Galunid**

### CODE REVIEW

**Comment by Hagb** [comment]
Great!
And please add shebang `#!/usr/bin/env python3` in the first line of executable `DeBooxUpx.py`

File: DeBooxUpx.py [hunk 1]
```diff
@@ -87,3 +87,16 @@ def deUpx(self, inputFileName: str, outputFileName: str):
...
         inputFile.close()
         outputFile.close()
+
+
+if __name__ == '__main__':
+    import sys
+    if len(sys.argv) >= 2:
+        device_name = sys.argv[1].capitalize()
+        updateUpxPath = "update.upx" if len(sys.argv) == 2 else sys.argv[2]
+        decryptedPath = updateUpxPath.replace(".upx", ".zip")
```
[the remark this answers is not in the document]
Comment (Galunid) [reply DeBooxUpx.py]: Should be good now!

File: DeBooxUpx.py [hunk 2]
```diff
@@ -87,3 +88,18 @@ def deUpx(self, inputFileName: str, outputFileName: str):
         self.deUpxSteam(inputFile, outputFile)
         inputFile.close()
         outputFile.close()
+
+
+if __name__ == '__main__':
+    import sys
+    if len(sys.argv) >= 2:
+        device_name = sys.argv[1].capitalize()
```
Comment (Hagb) [inline DeBooxUpx.py:96]: `capitalize()` may convert some device names to incorrect forms, for example `NovaPro` to `Novapro`

**Review by Hagb — CHANGES_REQUESTED** [review]

File: DeBooxUpx.py (hunk 2 above)
Comment (Galunid) [reply DeBooxUpx.py]: Done, I decided to print supported devices when incorrect key is passed, is that alright?

**Review by Hagb — APPROVED** [review]

File: DeBooxUpx.py (hunk 2 above)
Comment (Hagb) [reply DeBooxUpx.py]: Works well!

**Comment by Hagb** [comment]
Thanks for your contribution! It works well and is merged.

**Pull request merged**